bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueSensorHandler.java

```diff
@@ -145,7 +146,7 @@ public void dispose() {
                 return null;
             }
             ThingHandler handler = bridge.getHandler();
-            if (handler instanceof HueClient) {
+            if (handler instanceof HueBridgeHandler) {
```

Comment:
If you cast to `HueClient` later, the check for `HueClient` is correct. Why did you change it?

Reply:
I don't know why "HueClient" can't work, I try to change "HueClient" to "HueBridgeHandler" then it works. Can you help me check and debug.

Reply:
Mhm. Since `HueBridgeHandler` implements `HueClient` it should be ok.